sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_log_device_command_events_v1/metadata.yaml

```diff
@@ -20,7 +20,6 @@ labels:
 #   # so we must explicitly write out dependencies. In this case, the query
 #   # depends only on fxa logs produced via Stackdriver integration, so no other
 #   # scheduled tasks are involved and the referenced_tables list is empty.
-#   referenced_tables: []
```

Comment:
The preceding `# This query references secret keys...` comment should also be removed.